docker/Dockerfile

```diff
@@ -36,3 +36,4 @@ RUN apk add --no-cache --virtual .build-dependencies \
     apk del ${simulate_removal} .build-dependencies
 
 COPY search/ search/
+COPY example-search.json search.json
```

Comment:
Needed?

Reply:
We thought of two use cases:
- documentation for the format of the configuration file
- avoidance of exceptions in the application bootstrap in case it's run in isolation without providing a file

Reply:
For the former, not very useful as it's empty. For the latter, I'd expect it to fail... (hopefully with a useful message)

Reply:
this has been removed.